P1684-mdarray/P1684.md

```diff
@@ -884,14 +895,130 @@ constexpr mdarray(container_type&& c, const extents_type& ext);
 constexpr mdarray(container_type&& c, const mapping_type& m);
 ```
 
-[20]{.pnum} *Preconditions:* `c.size() == m.required_span_size()` is `true`.
+[20]{.pnum} *Preconditions:* `c.size() >= m.required_span_size()` is `true`.
 
 [21]{.pnum} *Effects:*
 
    * [21.1]{.pnum} Direct-non-list-initializes _`map_`_ with `m`, and
 
    * [22.2]{.pnum} Direct-non-list-initializes `ctr_` with `std::move(c)`.
 
+```c++
+template<class... SizeTypes>
```

Comment:
Do we need to consider the special case of `array` specializations here, as in the above constructors?

Reply:
no, this constructor requires constructibility from initializer list, i.e. its a forwarding constructor.